components/servo/Cargo.toml

```diff
@@ -103,6 +106,7 @@ surfman = { workspace = true, features = ["sm-angle-default"] }
 webxr = { path = "../webxr", optional = true }
 
 [target.'cfg(not(any(target_os = "android", target_env = "ohos")))'.dependencies]
+arboard = { workspace = true, optional = true }
```

Comment:
We enable the `clipboard` feature by default in in libservo, but `arboard` won't be available on Android and OHOS due to the conditional dependency here.